CHANGES.md

```diff
@@ -1,6 +1,7 @@
-# Unreleased
+# 2.13.0 (2020-04-14)
```

Comment:
Sorry didn't review this yesterday :)
Maybe update it for today, but it's not a big deal